src/devices/ilightsin.ts

```diff
@@ -10,16 +10,17 @@ export const definitions: DefinitionWithExtend[] = [
         extend: [m.light()],
     },
     {
-    zigbeeModel: ['13D'],
-    model: '13D',
-    vendor: 'Light',
-    description: 'iLightsin HSSA18-Z-MID zhaga module',
-    extend: [
-      m.deviceEndpoints({"endpoints":{"1":1,"33":33,"34":34}}),
-      m.light(),
-      m.illuminance({"endpointNames":["33"]}),
-      m.occupancy({"endpointNames":["34"]}),
-      m.iasZoneAlarm({"zoneType":"generic","zoneAttributes":["alarm_1","alarm_2","tamper","battery_low"]})],     
-    meta: {"multiEndpoint":true},
+        zigbeeModel: ["13D"],
+        model: "13D",
+        vendor: "Light",
+        description: "iLightsin HSSA18-Z-MID zhaga module",
+        extend: [
+            m.deviceEndpoints({endpoints: {"1": 1, "33": 33, "34": 34}}),
+            m.light(),
+            m.illuminance({endpointNames: ["33"]}),
+            m.occupancy({endpointNames: ["34"]}),
+            m.iasZoneAlarm({zoneType: "generic", zoneAttributes: ["alarm_1", "alarm_2", "tamper", "battery_low"]}),
```

Comment:
```suggestion
            m.iasZoneAlarm({zoneType: "occupancy", zoneAttributes: ["alarm_1", "alarm_2", "tamper", "battery_low"]}),
```

What kind of device is this? If occupancy use this.